### ISSUE TITLE
Suggestion: Track users card library

### ISSUE THREAD

**Issue by dantedog** [Epix37/Hearthstone-Deck-Tracker#699]
The tech to tell if a card fails to add to a decklist is already in the program. Could this be extended to add a series of preset decks (or a cleaner solution) containing every card and track cards not found? From there, I would imagine it wouldn't be to difficult to store the users cards, and highlight cards in decklists that are not found in the user's library.

**Comment by eXpressionist** [comment]
I'd love to see that! But it would take a lot of time to implement, and wouldn't be fully automated. I believe it's possible if ask user to create empty deck for every class to check all class cards, then check all common cards on the last hero.

**Comment by GiggliG** [comment]
The OP has a clever suggestion, with two problems I can think of.

1) Player card collection will change over time, so you need to be able to update with changes easily. (This could be automated using the same system).

2) The bigger problem is, some people like me play on more than one server, and others use different accounts. So people will have multiple different deck collections to track. This makes it complex. But not insurmountable - there could be a dropdown setting to switch which collection is active.

**Comment by dantedog** [comment]
I don't think this would ever be a fully automated thing. I would imagine that for the first run, it would take at least 10 inputs from the user. They would have to switch classes manually. 

After the first run, the user could manually trigger a re-scan (either full or only checking for cards not found in the user's library). Maybe also build in an option where you can manually add an individual card?

**Comment by culinko** [comment]
There is now Collection Tracker which tracks your collection. Check it out here: https://github.com/ko-vasilev/Hearthstone-Collection-Tracker/releases.
